### PR TITLE
Rebase trapezium

### PR DESCRIPTION
**Pull request by hutashan**
Please review this pull request. 
Please let me know if anything pending.

### CODE REVIEW

**Comment by debasish83** [comment]
Looks like it does not have all the code ?

**Comment by debasish83** [comment]
ohh ok...you can also do it for internal integration branch...I sync them up...

**Comment by debasish83** [comment]
Add back LuceneIndexer / DictionaryManager...your local branch was not updated with latest...

**Comment by hutashan** [comment]
@debasish83 and @rastogipankaj  . This is new rebase of trapezium. Please review it from your end.

**Comment by debasish83** [comment]
we can clean couchbase for now if needed...Redis seems to be getting more popular for kvstore...

File: cache/src/main/scala/com/verizon/bda/trapezium/cache/CommonCacheSessionFactory.scala [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+import scala.collection.mutable
+import scala.util.Try
+import org.slf4j.LoggerFactory
+
+/**
+  * Created by v468328 on 9/22/16.
+  * Common Cache Session Factory. It creates reference cache for application as per
+  * implementation specified in property commons.cache.impl.
+  * For CouchBase we need corresponding buckets in CouchBase.
+  * cbc.url= "http://146.1.180.119:8091/pools"
```
Comment (debasish83) [inline cache/src/main/scala/com/verizon/bda/trapezium/cache/CommonCacheSessionFactory.scala]: all these IP address need to be cleaned...v463 etc needs to be replaced with developer name

File: cache/target/analysis/compile.relations [hunk 2]
```diff
@@ -0,0 +1,27 @@
+products:
+   cache/src/main/scala/com/verizon/bda/trapezium/cache/CacheConfig.scala -> cache/target/scala-2.10/classes/com/verizon/bda/trapezium/cache/CacheConfig$.class
```
Comment (debasish83) [inline cache/target/analysis/compile.relations]: can you remove target...causing issues in review

File: cache/target/site/scaladocs/com/package.html [hunk 3]
```diff
@@ -0,0 +1,105 @@
+<?xml version='1.0' encoding='UTF-8'?>
```
Comment (debasish83) [inline cache/target/site/scaladocs/com/package.html]: the whole target folder can be cleaned

**Comment by debasish83** [comment]
target etc has been checked in...can we clean it up ?

**Comment by hutashan** [comment]
Deleted target folder.

File: dal/src/test/scala/com/verizon/bda/trapezium/dal/lucene/LuceneIndexerSuite.scala [hunk 4]
```diff
@@ -1,91 +0,0 @@
-package com.verizon.bda.trapezium.dal.lucene
-
```
Comment (debasish83) [inline dal/src/test/scala/com/verizon/bda/trapezium/dal/lucene/LuceneIndexerSuite.scala]: @hutashan please add LuceneIndexer and DictionaryManager from master...

File: framework/src/main/resources/couch.conf [hunk 5]
```diff
@@ -0,0 +1,12 @@
+cbc.url= "http://146.1.180.119:8091/pools"
```
Comment (debasish83) [inline framework/src/main/resources/couch.conf]: internal ip address should be cleaned

**Comment by hutashan** [comment]
Removed internal ip address.
Added Lucene from master

**Pull request closed without merging**